Sources/Sentry/SentryTracer.m

```diff
@@ -124,6 +127,8 @@ - (void)captureTransaction
 
     SentryTransaction *transaction = [[SentryTransaction alloc] initWithTrace:self children:spans];
     transaction.transaction = self.name;
+    [self addMeasurements:transaction];
```

Comment:
this should be sent only once though (App Start), we'd need to add a flag here to not do it twice